src/Sarif/Autogenerated/Location.cs

```diff
@@ -98,9 +98,9 @@ public Location()
         /// <param name="properties">
         /// An initialization value for the <see cref="P:Properties" /> property.
         /// </param>
-        public Location(Address address, PhysicalLocation physicalLocation, LogicalLocation logicalLocation, Message message, IEnumerable<Region> annotations, IDictionary<string, SerializedPropertyInfo> properties)
```

Comment:
>n(Address address, Phys [](start = 22, length = 23)

this seems like a bad change?

Reply:
oops, no this is correct! thanks for catching this!

---
In reply to: [268851294](https://github.com/Microsoft/sarif-sdk/pull/1367#discussion_r268851294) [](ancestors = 268851294)